unicore/webhooks/views.py

```diff
@@ -0,0 +1,85 @@
+from colander import (
+    MappingSchema, SchemaNode, String, Boolean, OneOf, url)
+
+from cornice.resource import resource, view
+
+from pyramid import httpexceptions
+from pyramid.view import view_config
+
+from sqlalchemy.orm.exc import NoResultFound, MultipleResultsFound
+
+from unicore.webhooks.models import DBSession, Webhook
+from unicore.webhooks.events import (
+    WebhookDeleted, WebhookCreated, WebhookUpdated)
+
+
+class WebhookSchema(MappingSchema):
+    url = SchemaNode(String(), location='body', validator=url)
+    event_type = SchemaNode(
+        String(), location='body',
+        validator=OneOf([value for value, label in Webhook.TYPES]))
+    active = SchemaNode(Boolean(), location='body')
+
+
+# Raise SQLAlchemy exceptions as HTTP 404s
+@view_config(context=NoResultFound)
+@view_config(context=MultipleResultsFound)
+def sqlachemy_error(exception, request):
+    raise httpexceptions.HTTPNotFound()
+
+
+@resource(collection_path='/hooks.json', path='/hooks/{uuid}.json')
```

Comment:
I think as a team we need to standardize on the format of our rest URLs. I think we should either use `/hooks.json` or `/hooks`, but we can't alternate between the 2.